<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolutionSettingsDialog</class>
 <widget class="QDialog" name="SolutionSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Solution settings</string>
  </property>
  <widget class="QWidget" name="ExperimentSettings" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>517</width>
     <height>68</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="6">
     <widget class="QLabel" name="labelExperimentSettings">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Experiment  settings:</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelPlanValidation">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Plan validation:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="PlanValidationEnabled">
      <property name="text">
       <string>Enabled</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="labelTimeout">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Timeout:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="TimeoutEnabled">
      <property name="text">
       <string>Enabled</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="labelTime">
      <property name="text">
       <string>time (s)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="TimeoutValue"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="PlannersSettings" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>83</y>
     <width>562</width>
     <height>297</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QLabel" name="labelPlanner">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Planner:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="labelIndividualTimeout">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Individual timeout:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QComboBox" name="PlannerSelected"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelParameters">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Parameters:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelPlatform">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Platform:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="7">
     <widget class="QLabel" name="labelPlannersSettings">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Planners settings:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="2">
     <widget class="QCheckBox" name="MacOS">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Mac OS</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QCheckBox" name="Windows">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Windows</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLabel" name="labelTime_2">
      <property name="text">
       <string>Time (s)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QCheckBox" name="Linux">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Linux</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="7">
     <widget class="QTableWidget" name="ParameterTable"/>
    </item>
    <item row="5" column="2" colspan="3">
     <widget class="QCheckBox" name="UseSpecificTimeout">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Use specific timeout</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QLineEdit" name="TimeoutValue_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>380</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SolutionSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
